doc/development_guide/run.rst

```diff
@@ -0,0 +1,83 @@
+===========================================
```

Comment:
Let's call this file `run_as_package` or something like that. `run.html` feels like we might want to use it for some more general pages in the future.